File: CS/WPFBlankDotNETAppWithNorthwindDatabase/Models/Territory.cs
using System;
using System.Collections.Generic;

namespace WPFBlankDotNETAppWithNorthwindDatabase.Models;

public partial class Territory
{
    public string TerritoryId { get; set; }

    public string TerritoryDescription { get; set; }

    public int RegionId { get; set; }

    public virtual Region Region { get; set; }
}

File: VB/WPFBlankDotNETAppWithNorthwindDatabase/Models/Territory.vb
Imports System
Imports System.Collections.Generic

Namespace WPFBlankDotNETAppWithNorthwindDatabase.Models

	Partial Public Class Territory
		Public Property TerritoryId() As String

		Public Property TerritoryDescription() As String

		Public Property RegionId() As Integer

		Public Overridable Property Region() As Region
	End Class
End Namespace
